homotopy-model/src/model/proof.rs

```diff
@@ -327,6 +323,7 @@ impl ProofState {
                 .workspace
                 .as_ref()
                 .map_or(false, |ws| ws.visible_dimension() > 0),
+            Action::UpdateView(v) => v.dimension <= 4,
```

Comment:
I don't understand why we need this. Views are constrained to 0-4 (see their constructor).

Reply:
Actually this reminds me that I would like View to be an enum. I don't remember why I never did that.